doc/modules/classes.rst

```diff
@@ -143,43 +143,58 @@ Classes
    covariance.oas
    covariance.graph_lasso
 
+.. _model_selection_ref:
```

Comment:
I'm also convinced now that we should not list them here. Maybe just not list them. People can always look at the docs in ipython or the source if they like.